test/helpers.bash

```diff
@@ -512,4 +512,11 @@ function dig_reverse() {
 # third arg is server addr
     #run_in_container_netns "$1" "dig" "-x" "$2" "+short" "@$3"
     run_in_container_netns "$1" "nslookup" "$2" "$3"
+}
+
+#
+# Pops a number of arguments from CONTAINER_CONFIGS
+#
+function remove_container_configs() {
+    unset CONTAINER_CONFIGS[${#CONTAINER_CONFIGS[@]}-$1]
 }
```

Comment:
Missing newline

Reply:
I removed this function in favor of just resetting the array as you suggested.